src/Common/Markdown/MarkDown.tsx

```diff
@@ -0,0 +1,100 @@
+import { marked } from 'marked'
+import DOMPurify from 'dompurify'
+import { useEffect, useRef } from 'react'
+
+const MarkDown = (
+    setExpandableIcon: (arg0: boolean) => void,
+    markdown = '',
+    className = '',
+    breaks = false,
+    disableEscapedText = false,
+    ...props: any[]
```

Comment:
- add types, also create a separate file for types
- We are not exporting the component